apps/a11y-tests/src/tests/__snapshots__/ComponentExamples.test.tsx.snap

```diff
@@ -1,6836 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
pls revert this file

Reply:
other than that, the change looks good to me. `SpinButton` has always been relying on `Fabric` wrapper to drop the default fonts, which I think we should change to let `SpinButton` to define fonts on it own (your change)

Reply:
Thanks @xugao, done!

Reply:
looks like there are some snapshots you missed to update. To do that, go to `packages/react-examples` and run `yarn update-snapshots`

Reply:
Sounds good, thanks @xugao! Updated the snapshots.